src/xmipp/libraries/data/blobs.cpp

```diff
@@ -929,8 +929,8 @@ void blobs2voxels(const GridVolume &vol_blobs,
     {
         int min_distance = (int)ceil((2*(vol_blobs.grid(i)).relative_size ) / blob.radius ) + 1;
 
-        slices_status = (int *)malloc(sizeof(int)*(int)((ZZ((&(vol_blobs.grid(i)))->highest)-ZZ((&(vol_blobs.grid(i)))->lowest)+1)));
-        memset(slices_status,0,sizeof(int)*(int)((ZZ((&(vol_blobs.grid(i)))->highest)-ZZ((&(vol_blobs.grid(i)))->lowest)+1)));
+        slices_status = (int *)malloc(sizeof(int)*(int)(ZZ((&(vol_blobs.grid(i)))->highest)-ZZ((&(vol_blobs.grid(i)))->lowest)+1));
```

Comment:
Absolutely not.
Code like this has to be refactored.
If you cannot think of anything better, at least use `new int[some_size]()` (notice the `()` at the end, to zero-initialize it (i.e. you also avoid the `memset` call under)